transport.go

```diff
@@ -49,6 +50,20 @@ type accessToken struct {
 	Repositories []github.Repository            `json:"repositories,omitempty"`
 }
 
+// RefreshTokenError represents a custom error for the refresh access tokens operations.
+// It enables the caller to inspect the root cause and response.
+type RefreshTokenError struct {
+	Message        string
+	RootCause      error
+	InstallationID int64
+	Request        *http.Request
```

Comment:
We can omit `Request` here, since it's already accessible through `Response.Request`.

Reply:
Fixed in https://github.com/bradleyfalzon/ghinstallation/pull/45/commits/dc25674ef598f02df854cc1ce52f4dd656984f13